src/Compilers/CSharp/Portable/FlowAnalysis/NullableWalker.cs

```diff
@@ -1811,6 +1811,16 @@ public override BoundNode VisitObjectCreationExpression(BoundObjectCreationExpre
             return null;
         }
 
+        public override BoundNode VisitUnboundObjectCreationExpression(UnboundObjectCreationExpression node)
```

Comment:
>VisitUnboundObjectCreationExpression [](start = 34, length = 36)

I'm surprised. Since LocalRewriter throws when reaching an `UnboundObjectCreationExpression`, I thought this node doesn't survive initial binding, but if that's the case then `NullableWalker` woudn't see it.

Reply:
I looked at the switch expression to see how it manages to not get here, turns out it actually does but we bind it with an ErrorType, hence the conversion is unreachable in VisitConversion.

The weird part is that it seems like the nullable walker is ran twice, at the second time we reach here.

Reply:
I looked at the switch expression to see how it manages to not get here, turns out it actually does but we bind it with an ErrorType, hence the conversion is unreachable in VisitConversion.

The weird part is that it seems like the nullable walker is ran twice, at the second time we reach here.